app/src/main/res/values/splash.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Theme.Splash" parent="Theme.SplashScreen">
+        <item name="windowSplashScreenBackground">@color/Neutral_01_black</item>
+        <item name="windowSplashScreenAnimatedIcon">@drawable/ic_splash</item>
+        <item name="windowSplashScreenAnimationDuration">4000</item>
```

Comment:
this duration is only for animation, it doesn't even works 
![image](https://github.com/DavitTsiskarashvili/Movies/assets/95241918/0bbb03ed-874c-432a-a9e8-0e75882d1f39)
https://developer.android.com/develop/ui/views/launch/splash-screen

Reply:
done